credoai/evaluators/identity_verification.py

```diff
@@ -54,11 +59,58 @@ class IdentityVerification(Evaluator):
             subject: it means if any pairs of samples for the same subject are a match, the subject pair
                 is marked as a match. Some identity verification use cases improve overall accuracy by storing
                 multiple samples per identity. Subject-level comparison mirrors this behavior.
+
+    Example
+    --------
+    import pandas as pd
```

Comment:
Beautiful! I will use this as an example for the work on Evaluator documentation!